usr/log.c

```diff
@@ -336,6 +336,20 @@ void log_info(const char *fmt, ...)
 	va_end(ap);
 }
 
+void log_session_reopen(int level, iscsi_session_t *session, const char *fmt, ...)
+{
+	int reopen_cnt = session->reopen_cnt;
+	int reopen_log_freq = session->reopen_log_freq;
+
+	if ((log_level >= level) && ((log_level > 2) ||
+		(reopen_cnt % reopen_log_freq) == 0)) {
```

Comment:
so this prints a message every time, even if debug logging isn't enabled, if the freq is 1? And if it's say, above 1, like e.g. 10, you would get a message every tenth time, plus every other time if debug logging is enabled?

That makes my head hurt ...

Reply:
Yes, you are right. Changed to print only if the debug logging is enabled.